test/activities/CategoriesEntity.js

```diff
@@ -78,7 +80,7 @@ describe('CategoriesEntity', () => {
 	});
 
 	describe('non editable', () => {
-		it('sets canEditName to true', () => {
+		it('sets canEditCategories to true', () => {
```

Comment:
Still need to update `true` to `false` in this test name, based on what it's doing.